### DISCUSSION TITLE
logout from i3, when I don't have keybindings

### DISCUSSION THREAD

**Discussion by lalitmee** [i3/i3#5471] [Q&A] [answered]
I installed i3 on a new system which was having Ubuntu 20.04 and I enabled auto login in Ubuntu. Now I use `sxhkd` for my keybindings but its not installed due to which my keybindings are not working in i3 and I can't even open terminal or anything else for that matter. If I restart my system it goes into i3 directly as there is auto-login enabled.

How can I logout from i3 so that I can fix these by going into Ubuntu and come back again?

Please help me. This is my work laptop and I don't want to mess this up.

**Comment by okraits** [comment]
If you are looking for the default keybinding for exiting i3, it is Mod1+Shift+e.

**Comment by orestisfl** [comment, marked as answer]
Ctrl+Alt+F2 should drop you to a new tty. From there you can login to a new shell. You can edit your config there and install missing dependencies.

**Reply by lalitmee** [reply]
I disabled auto login using tty and then got back to ubuntu. Then installed everything and it worked fine.
